lib/helpers.nix

```diff
@@ -86,8 +86,10 @@ rec {
     );
 
     mkInt = default: mkNullable lib.types.int (toString default);
-    mkBool = default: mkNullable lib.types.bool (toString default);
+    mkBool = default: mkNullable lib.types.bool (if default then "true" else "false");
```

Comment:
why this change?

Reply:
nvm, got it, `toString true/false` returns `1/0`

Reply:
Worse than that, it returns "1"/""